app/controllers/application_controller.rb

```diff
@@ -182,15 +182,17 @@ def error_handler(e)
             _("Action not implemented")
           when ::AbstractController::ActionNotFound # Prevent Rails showing all known controller actions
             _("Unknown Action")
+          when ::MiqException::RbacPrivilegeException
+            _("The user is not authorized for this task or item")
           else
             e.message
           end
 
-    render_exception(msg)
+    render_exception(msg, e)
```

Comment:
I looked... this is the only place the `render_exception` is called in any of the repos so it's safe that you changed the interface.